dotCMS/src/main/java/com/dotcms/api/system/event/SystemEventType.java

```diff
@@ -26,6 +26,8 @@
  */
 public enum SystemEventType {
 
+	LOCAL_SYSTEM_EVENT,
```

Comment:
This should be `CLUSTER_WIDE_EVENT`   I would also put javadocs on each of the enum properties